### DISCUSSION TITLE
What do errors like `ValueError: No metadata file found for the following package: cython` mean?

### DISCUSSION THREAD

**Discussion by Vuizur** [pyodide/pyodide#4080] [Q&A] [answered]
When following the steps [here](https://pyodide.org/en/stable/development/new-packages.html) and running the build-recipes command, I get similar errors.

I assume that it means that no pure Python wheel was found and that the package was also was not in the packages folder so that I should perform the same steps as detailed in the linked docs for the package mentioned in the error? 

Thanks for the help!

**Comment by ryanking13** [comment, marked as answer]
Could you share us the `meta.yaml` file you created? The error message means that you specified `cython` in `requirements`, but `cython` should not be in the `requirements`.

**Reply by Vuizur** [reply]
Thanks for the answer! 

Have I understood this correctly: When porting meta.yaml from conda to pyodide, one should generally delete packages such as `cython`, `python`, `pip`, `wheel`, `setuptools`, `typing_extensions`  from the requirements section?

**Reply by ryanking13** [reply]
Right. In most cases, you only need to specify python packages in `requirements.run`.
